adapters/rubicon/rubicon.go

```diff
@@ -785,7 +802,8 @@ func (a *RubiconAdapter) MakeRequests(request *openrtb.BidRequest, reqInfo *adap
 			}
 
 			videoCopy := *thisImp.Video
-			videoExt := rubiconVideoExt{Skip: rubiconExt.Video.Skip, SkipDelay: rubiconExt.Video.SkipDelay, VideoType: videoType, RP: rubiconVideoExtRP{SizeID: rubiconExt.Video.VideoSizeID}}
+			sizeId := resolveVideoSizeId(thisImp.Video.Placement, thisImp.Instl)
+			videoExt := rubiconVideoExt{Skip: rubiconExt.Video.Skip, SkipDelay: rubiconExt.Video.SkipDelay, VideoType: videoType, RP: rubiconVideoExtRP{SizeID: sizeId}}
```

Comment:
Given that the `rubiconExt.Video.VideoSizeID` field will no longer be used here nor elsewhere, should we remove the check on lines 791 to 796 above? 
```
790         if isVideo {
791 -           if rubiconExt.Video.VideoSizeID == 0 {
792 -               errs = append(errs, &errortypes.BadInput{
793 -                   Message: fmt.Sprintf("imp[%d].ext.bidder.rubicon.video.size_id must be defined for video impression", i),
794 -               })
795 -               continue
796 -           }
797
798             // if imp.ext.is_rewarded_inventory = 1, set imp.video.ext.videotype = "rewarded"
799             var videoType = ""
800             if bidderExt.Prebid != nil && bidderExt.Prebid.IsRewardedInventory == 1 {
801                 videoType = "rewarded"
802             }
803
804             videoCopy := *thisImp.Video
805             sizeId := resolveVideoSizeId(thisImp.Video.Placement, thisImp.Instl)
806             videoExt := rubiconVideoExt{Skip: rubiconExt.Video.Skip, SkipDelay: rubiconExt.Video.SkipDelay, VideoType: videoType, RP: rubiconVideoExtRP{SizeID: sizeId}}
807             videoCopy.Ext, err = json.Marshal(&videoExt)
adapters/rubicon/rubicon.go
```
If future imps come with an empty `VideSizeID` (JSON field `size_id`) field, we might risk wrongfully discarding impressions.